x-pack/plugins/canvas/canvas_plugin_src/uis/arguments/datacolumn/simple_math_function.js

```diff
@@ -8,29 +8,23 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { EuiSelect } from '@elastic/eui';
 
-export const SimpleMathFunction = ({ onChange, value, inputRef, onlymath }) => {
-  const options = [
-    { text: 'Average', value: 'mean' },
-    { text: 'Count', value: 'size' },
-    { text: 'First', value: 'first' },
-    { text: 'Last', value: 'last' },
-    { text: 'Max', value: 'max' },
-    { text: 'Median', value: 'median' },
-    { text: 'Min', value: 'min' },
-    { text: 'Sum', value: 'sum' },
-    { text: 'Unique', value: 'unique' },
-  ];
+const options = [
+  { text: 'Average', value: 'mean' },
+  { text: 'Count', value: 'size' },
+  { text: 'First', value: 'first' },
+  { text: 'Last', value: 'last' },
+  { text: 'Max', value: 'max' },
+  { text: 'Median', value: 'median' },
+  { text: 'Min', value: 'min' },
+  { text: 'Sum', value: 'sum' },
+  { text: 'Unique', value: 'unique' },
+];
```

Comment:
Moved this outside the component so we don't need to recreate the array every time this component gets rendered.